.github/ISSUE_TEMPLATE/011-bug-results.yml

```diff
@@ -0,0 +1,98 @@
+name: Bug (model evaluation)
+description: Something goes wrong when evaluating a model without any complex components such as the server on top.
+title: "Eval bug: "
```

Comment:
I agree that it makes sense to use language that regular users can understand. I think however that we should avoid the term "quality" since a lot of the things we do for better performance or less memory use (such as quantization) trade in some model quality. I also want to contrast this template with a future template for "model training". I went with the term "model use" instead.